### PR TITLE
Move clru repo

### PR DESCRIPTION
**Pull request by tzulberti-jampp**

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Claudio Freire <klaussfreire@gmail.com>
```
Comment (klaussfreire) [inline AUTHORS.rst:8]: Maybe add yourself as contributor

File: LICENSE [hunk 2]
```diff
@@ -1,165 +1,32 @@
-                   GNU LESSER GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
- Everyone is permitted to copy and distribute verbatim copies
- of this license document, but changing it is not allowed.
 
+BSD License
```
Comment (klaussfreire) [inline LICENSE]: You changed the license. Not cool.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.. image:: https://img.shields.io/travis/klaussfreire/clru.svg
+        :target: https://travis-ci.org/klaussfreire/clru
+
+.. image:: https://readthedocs.org/projects/clru/badge/?version=latest
+        :target: https://clru.readthedocs.io/en/latest/?badge=latest
+        :alt: Documentation Status
+
+
+Cython LRU Structures
+* Free software: BSD license
```
Comment (klaussfreire) [inline README.rst]: Use the right license here

File: requirements-doc.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+Sphinx==1.8.5
+sphinx-rtd-theme==0.4.3
+sphinx-markdown-tables==0.0.9
+recommonmark==0.5.0
+doc8==0.8.0
+sphinxcontrib-httpdomain==1.7.0
```
Comment (klaussfreire) [inline requirements-doc.txt]: Should these really be pinned?

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# The depenencies must not be fixed to a specific version, it you might
+# limit the min version required. For example: 
+#
+#   do this:
+#       six>=1.0.0
+#       
+#   don't do this:
+#       six==1.0.0
+#       libcore==1.2.3
```
Comment (klaussfreire) [inline requirements.txt]: Remove the reference to libcore

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+import os
+
+import sys
+
+
```
Comment (klaussfreire) [inline setup.py]: Check blank lines

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+from Cython.Build import cythonize
+
+from setuptools import setup, find_packages
+
+
+setup_requires = []
+
+setup_requires.append("pytest-runner==5.1")
+
+
```
Comment (klaussfreire) [inline setup.py]: Check blank lines

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+from setuptools.command.build_ext import build_ext
+from Cython.Distutils import build_ext as cython_build_ext, Extension as CythonExtension
+from Cython.Build import cythonize
+
+from setuptools import setup, find_packages
+
+
+setup_requires = []
+
+setup_requires.append("pytest-runner==5.1")
```
Comment (klaussfreire) [inline setup.py]: Shouldn't this be test requires?

File: setup.py [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+            for line in requirements
+        ]
+        # remove empty lines
+        requirements = list(filter(None, requirements))
+        # remove whitespaces
+        requirements = [line.strip().replace(" ", "") for line in requirements]
+        return requirements
+
+
+
```
Comment (klaussfreire) [inline setup.py]: Check blank lines

File: setup.py [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+                pool.map(self.build_extension, self.extensions, chunksize=1)
+                pool.close()
+                pool.join()
+        cmd_class["build_ext"] = parallel_build_ext
+
+
+
+
+
+
```
Comment (klaussfreire) [inline setup.py]: Check blank lines

File: setup.py [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+setup(
+    name="clru",
+    description="Cython LRU Structures",
+    version=version,
+    author="Claudio Freire",
+    author_email="klaussfreire@gmail.com",
+    classifiers=[
+        "License :: OSI Approved :: BSD License",
```
Comment (klaussfreire) [inline setup.py]: Use proper license

File: setup.py [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+    long_description=readme + "\n\n" + history,
+    long_description_content_type="text/x-rst",
+    package_data={
+        "": ["*.pxd", "*.pyx"],
+        "clru": ["VERSION"],
+    },
+    packages=find_packages(include=["clru", "clru.*"]),
+    setup_requires=setup_requires,
+    test_suite="tests",
+    tests_require=["pytest==4.6.3", ],
```
Comment (klaussfreire) [inline setup.py]: Don't pin it

**Review by klaussfreire — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+
+Sort subsections like so: Added, Bugfixes, Improvements, Technical tasks.
+Group anything an end user shouldn't care deeply about into technical
+tasks, even if they're technically bugs. Only include as "bugfixes"
+bugs with user-visible outcomes.
+
+When major components get significant changes worthy of mention, they
+can be described in a Major section.
+
+More information can be found HERE:
```
Comment (klaussfreire) [inline CHANGELOG.rst]: Looks like it's missing a link

**Review by klaussfreire — CHANGES_REQUESTED** [review]

File: requirements-doc.txt (hunk 4 above)
Comment (klaussfreire) [reply requirements-doc.txt]: Ping

File: README.rst (hunk 3 above)
Comment (klaussfreire) [reply README.rst]: This

File: requirements-doc.txt (hunk 4 above)
Comment (tzulberti-jampp) [reply requirements-doc.txt]: Yes, I prefer to fix the sphinx and other libraries versions.. They are doing some work with Sphinx 3.0.0, so I prefer to pin the documentation libraries

Comment (klaussfreire) [reply requirements-doc.txt]: Why not use `<3.0.0` then? Accept patches, but reject major updates

**Review by klaussfreire — APPROVED** [review]

**Pull request merged**